### ISSUE TITLE
patch generation

### ISSUE THREAD

**Issue by tofutim** [charlesw/tesseract-vs2012#16]
how is the patch generated? would it be better to just have a fork off of tesseract?

**Comment by charlesw** [comment]
Yes a fork would be easier, this repo was created before tesseract moved to
github which is why a patch was used.

**Comment by tofutim** [comment]
should I be using the 'git' patch?
